Samplero/Samplero/Screen/Camera/CameraViewController.swift

```diff
@@ -0,0 +1,260 @@
+        photoHistoryButton.addTarget(self, action: #selector(didTapHistory), for: .touchUpInside)
+        cartButton.addTarget(self, action: #selector(didTapCart), for: .touchUpInside)
+    }
+    
+    private func checkCameraPermissions() {
+        switch AVCaptureDevice.authorizationStatus(for: .video) {
+        case .notDetermined:
+            // Request
+            AVCaptureDevice.requestAccess(for: .video) { granted in
+                guard granted else {
+                    return
+                }
+                DispatchQueue.main.async {
+                    self.setUpCamera()
+                }
+            }
+        case .restricted:
+            break
+        case .denied:
+            break
+        case .authorized:
+            setUpCamera()
+        @unknown default:
+            break
+        }
+    }
+    
+    private func setUpCamera() {
+        let session = AVCaptureSession()
+        if let device = AVCaptureDevice.default(for: .video) {
+            do {
+                let input = try AVCaptureDeviceInput(device: device)
+                if session.canAddInput(input) {
+                    session.addInput(input)
+                }
+                
+                if session.canAddOutput(output) {
+                    session.addOutput(output)
+                }
+                
+                previewLayer.videoGravity = .resizeAspectFill
+                previewLayer.session = session
+                
+                DispatchQueue.global().async {
+                    session.startRunning()
+                }
+                self.session = session
+            } catch {
+                print(error)
+            }
+        }
+    }
+    
+    @objc private func didTapTakePhoto() {
+        print("clicked take photo button")
+        
+        #if !targetEnvironment(simulator)
+        output.capturePhoto(with: AVCapturePhotoSettings(),
+                            delegate: self)
+        #endif
+    }
+    
+    @objc private func didTapBringPhoto() {
+        // TODO: bring photo from library
+        print("clicked bring photo from library")
+    }
+    
+    @objc private func didTapHistory() {
+        // TODO: open history
+        print("clicked history")
+    }
+    
+    @objc private func didTapCart() {
+        // TODO: open cart
+        print("clicked cart")
+    }
+}
+
+// MARK: - Extension
+
+extension CameraViewController: AVCapturePhotoCaptureDelegate {
+    func photoOutput(_ output: AVCapturePhotoOutput, didFinishProcessingPhoto photo: AVCapturePhoto, error: Error?) {
+        guard let data = photo.fileDataRepresentation() else {
+            return
+        }
+        let image = UIImage(data: data)
+        
+        session?.stopRunning()
+        
+        let imageView = UIImageView(image: image)
+        imageView.contentMode = .scaleAspectFill
+        imageView.frame = view.bounds
+        view.addSubview(imageView)
```

Comment:
p5.

이건 궁금증인데

<img width="480" alt="image" src="https://user-images.githubusercontent.com/59143479/194752311-9d35f4bf-688d-4d9c-9d91-3c4794cc5443.png">

위처럼 사진을 `topDrawer` 의 bottom과 `bottomDrawer`의 top에 맞춰주려고 다음과 같이 작성했는데 의도치 않은 결과가 나올까요

```swift
        imageView.snp.makeConstraints {
            $0.top.equalTo(topDrawer.snp.bottom)
            $0.bottom.equalTo(bottomDrawer.snp.top)
            $0.leading.trailing.equalToSuperview()
        }
```

![IMG_2443](https://user-images.githubusercontent.com/59143479/194752459-515dd8cf-1aa3-4d17-ac9a-22c62e2683a5.PNG)

cc. @KimDaeSeong8721

Reply:
![image](https://user-images.githubusercontent.com/69894461/194773301-519b5e5b-79b1-4371-a5d7-379281659aa5.png)
이런 것을 원하신거라면... 
```suggestion
       imageView.layer.zPosition = -1
        view.addSubview(imageView)

```

Reply:
추가적으로 autolayout으로 했을때 코지가 했던것처럼 보인 이유는 
` imageView.contentMode = .scaleAspectFill `때문이네요. 비율에 맞게 채워주기 위해서 침범하네요,,
scaletoFill로 바꾸면 딱맞게 들어갑니다. 다만 비율이 달라지네요. 
(그리고 UIImageView 안붙여도 PreViewLayer에 찍은 사진이 바로 보이네여 띠용)

Reply:
아하 이미지 비율맞추려고 그랬던거군요 bb

Reply:
> (그리고 알아낸건데 UIImageView 안붙여도 PreViewLayer에 찍은 사진이 바로 보이네여 띠용)

띠용